packs/elastic/scripts/logsfeed.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python
+# encoding: utf-8
+import sys
+import os
+import datetime
+import time
+import random
+import socket
+from pprint import pprint
+from random import choice
+from jinja2 import Template
+
```

Comment:
Short module level docstring on what this script does and how to use it would be nice :)